tests/test_approximations.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from jaxkern.approximations import RFF
+from jaxkern.stationary import (
+    Matern12,
+    Matern32,
+    Matern52,
+    RBF,
+    RationalQuadratic,
+    PoweredExponential,
+    Periodic,
+)
+from jaxkern.nonstationary import Polynomial, Linear
+from jaxkern.base import AbstractKernel
+import jax.random as jr
+from jax.config import config
+import jax.numpy as jnp
+from jaxlinop import DenseLinearOperator
+from typing import Tuple
+import jax
+
+config.update("jax_enable_x64", True)
+_jitter = 1e-5
+
+
+@pytest.mark.parametrize("kernel", [RBF, Matern12, Matern32, Matern52])
+@pytest.mark.parametrize("num_basis_fns", [2, 10, 20])
+@pytest.mark.parametrize("n_dims", [1, 2, 5])
+def test_frequency_sampler(kernel: AbstractKernel, num_basis_fns: int, n_dims: int):
+    key = jr.PRNGKey(123)
+    base_kernel = kernel(active_dims=list(range(n_dims)))
+    approximate = RFF(base_kernel, num_basis_fns)
+
+    params = approximate.init_params(key)
+    assert params["frequencies"].shape == (num_basis_fns, n_dims)
```

Comment:
At no point do we test that the right spectral density is coming out of the kernels. Should we?

i.e. rbf -> Normal e.t.c.

Reply:
Good idea - added as a test.